private-compressed-account/tests/private-compressed-account.ts

```diff
@@ -46,35 +45,41 @@ describe("Test private-compressed-account", () => {
     }
   });
 
-  it("Functional Test Inclusion Gt Circuit", async () => {
-    const compressedAccount = new PoseidonCompressedAccount(POSEIDON, IDL, 0);
-    let insertValue = "12";
+  let compressedAccount: PoseidonCompressedAccount;
+  let insertValue = "12";
+
+  it("Inclusion Gt Circuit should succeed", async () => {
+    compressedAccount = new PoseidonCompressedAccount(POSEIDON, IDL, 0);
     let leafHash = POSEIDON.F.toString(POSEIDON([insertValue]));
     await compressedAccount.generateUpdateProof({ leafHash });
 
+    log(`insertValue 12, refValue 12`);
```

Comment:
We have no strong typescript guidelines, so it's just an opinion.
I would prefer to use single or double quotes here, not backticks, because it's just a static string, not template.

Btw, I think we should consider to use more strict approach, like this https://google.github.io/styleguide/jsguide.html#features-strings-template-strings and this https://google.github.io/styleguide/tsguide.html